### DISCUSSION TITLE
StyleTTS2 produces flawless english speech, can it be added as option into this repo  ?

### DISCUSSION THREAD

**Discussion by cmp-nct** [coqui-ai/TTS#3272] [Ideas]
I played a bit with the recently released StyleTTS2 and it appears superior in english to anything I've seen before https://github.com/yl4579/StyleTTS2
It also supports a ton of configuration options to adapt the speech to your needs, it also was very quick in synthesizing.
It's MIT licensed, model is included for free.
The only downfall is currently no multi-language support, though given the quality it produces that might change.

Configuration options: https://github.com/yl4579/StyleTTS2/blob/main/Demo/Inference_LibriTTS.ipynb

**Comment by erogol** [comment]
Sure it can be added.  We don't have time for it internally but I can guide if there is someone interested.

**Reply by RootingInLoad** [reply]
Would definitely be interested if the offer still stands.

**Reply by erogol** [reply]
Sure!! can you DM me on Discord?

**Reply by DrewThomasson** [reply]
HYPED FOR THIS FEATURE

**Reply by cmp-nct** [reply]
They are heavily working on multilanguage support already, looks very promising!
